com.openmusicplayer.OMP.json

```diff
@@ -0,0 +1,257 @@
+{
+
+  "app-id": "com.openmusicplayer.OMP",
+
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+
+  "sdk": "org.gnome.Sdk",
+
+
```

Comment:
You didn't care to change the name of icon and desktop files, and you didn't provide [AppData](https://github.com/flathub/flathub/wiki/AppData-Guidelines).

Familiarize yourself with the options below:
- `rename-appdata-file`
- `rename-desktop-file`
- `rename-icon`
- `copy-icon`

Ideally, these changes should be upstreamed.

Reply:
Nevermind, it looks like it's actually handled by Makefile.
https://github.com/TheWiseNoob/OMP/blob/680ebbbfca1be7633982c07264ff95f1d54e3997/Makefile#L535-L541
However, mode for icons should be `0644` instead of `0755`.
Also, fixed paths are not a very good idea. Hint:
https://www.gnu.org/software/autoconf/manual/autoconf.html#Installation-Directory-Variables

Anyway, the AppData file is still required.

Reply:
Thank you for all the tips!